test/integrated/lsms.jl

```diff
@@ -151,9 +151,11 @@ for FT in (Float32, Float64)
             m::DummyModel3{FT},
         ) where {FT}
             update_aux! = ClimaLSM.make_update_aux(m)
+            update_boundary_fluxes! = ClimaLSM.make_update_boundary_fluxes(m)
```

Comment:
should `update_aux` for `DummyModel4` also call `update_boundary_fluxes`?

Reply:
Right now, `update_aux` is a separate function than `update_boundary_fluxes`. They both update parts of the cache, but not the same variables.